symplyphysics/laws/electricity/circuits/resistance_of_the_serial_RLC_circuit.py

```diff
@@ -0,0 +1,53 @@
+from sympy import (Eq, solve, sqrt,)
+from symplyphysics import (units, Quantity, Symbol, Vector, print_expression, validate_input,
+    validate_output, vector_magnitude,)
+from symplyphysics.core.expr_comparisons import expr_equals
+
+# Description
+## Consider an electrical circuit consisting of a capacitor, coil, and resistor connected in series.
+## Then you can find the total resistance of such a circuit.
+
+## Law is: R = sqrt(R0^2 + (Xl - Xc)^2), where
+## R - resistance of circuit,
+## R0 - resistance of resistor
+## Xl - inductive resistance,
+## Xc - capacitive resistance.
+
+circuit_resistance = Symbol("circuit_resistance", units.impedance)
+
+resistance_resistor = Symbol("resistance_resistor", units.impedance)
+capacitive_resistance = Symbol("capacitive_resistance", units.impedance)
+inductive_resistance = Symbol("inductive_resistance", units.impedance)
+
+law = Eq(circuit_resistance, sqrt(resistance_resistor**2 + (inductive_resistance - capacitive_resistance)**2))
+
+# This law might be derived via impedance definition. The impedance is Z = R0 + j * X.
+# Then the resistance of the circuit is equal to the impedance modulus and is equal to sqrt(R0^2 + X^2).
+# X = Xc - Xl
+# Then, as a result, the resistance of the circuit is equal to sqrt(R0^2 + (Xl - Xc)^2).
+
+# The complex impedance can be represented as a vector with two components: the real and imaginary parts.
+# Let's find the modulus of this vector and get the resistance of the circuit.
+
+resistance_vector = Vector([resistance_resistor, inductive_resistance - capacitive_resistance])
```

Comment:
We do not know, that inductive and capacitive reactances have different signs. Try to find and add relevant law to justify it.

Reply:
> We do not know, that inductive and capacitive reactances have different signs. Try to find and add relevant law to justify it.

I can use these two laws:
coil_impedance_from_inductivity_and_frequency
capacitor_impedance_from_capacitance_and_frequency

But in this case, this law will change its form.
Before:
R = sqrt(R0^2 + (Xl - Xc)^2)
After:
R = sqrt(R0^2 + (w * L - (1/(w * C))^2)

Reply:
No, you should not change the law.
Add laws that `w * L` is `Xl` and `1 / (w * C)` is `Xc`
From these laws you'll be able to justify `Xl - Xc`.
You also need to justify the sum of reactances. I suggest to add law that impedance of serial circuit is a sum of its impedances. And substitute impedance of each element to get the resulting formula.

Reply:
> No, you should not change the law. Add laws that `w * L` is `Xl` and `-1 / (w * C)` is `Xc` From these laws you'll be able to justify `Xl - Xc`. You also need to justify the sum of reactances. I suggest to add law that impedance of serial circuit is a sum of its impedances. And substitute impedance of each element to get the resulting formula.

> I suggest to add law that impedance of serial circuit is a sum of its impedances
I understand that. I'll just add the law with the equation shown below:
Eq(impedance, SumArray(impedances))

> Add laws that `w * L` is `Xl` and `-1 / (w * C)` is `Xc` From these laws you'll be able to justify `Xl - Xc`.
I don't understand that. Is it necessary to add a laws with such an equation?
1) Eq(w * L, Xl)
2) Eq(-1 / (w * C), Xc)
If so, then this is no different from the equations in the existing laws:
coil_impedance_from_inductivity_and_frequency Eq(coil_impedance, I * circular_frequency * coil_inductivity))
capacitor_impedance_from_capacitance_and_frequency Eq(capacitor_impedance, -I / (circular_frequency * capacitor_capacitance))

Reply:
> If so, then this is no different from the equations in the existing laws
They are different. From existing laws you cannot explain reactance. So basically `Xl` and `Xc` are not covered by existing laws. We can keep it this way and avoid using `reactance` at all. Or we can define them and link with existing laws.

Reply:
Another option is to use https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/definitions/impedance_is_resistance_and_reactance.py, assume resistance of coil and capacitor as `0` and derive their reactance. In this case we won't need to define their reactances.